tests/gtest_like_c.c

```diff
@@ -0,0 +1,31 @@
+#include "gtest_like_c.h"
+
+jmp_buf gtest_jmp;
+
+int gtest_like_c_run_tests(const struct Test* tests, size_t num_tests) {
```

Comment:
Did you consider using some C-only unit testing framework like [https://www.throwtheswitch.org/unity](https://www.throwtheswitch.org/unity) ?

Reply:
my personal preferences are to reuse gtest and return back to gtest once C++ is ready or by integrating clang